ci/jenkins/test.sh

```diff
@@ -120,6 +120,11 @@ export NO_PULL
 
 E2ETEST_PATH=${WORKDIR}/kubernetes/_output/dockerized/bin/linux/amd64/e2e.test
 
+if [[ $TESTCASE =~ "multicast" ]]; then
+    CONTROL_NODE=$(kubectl get nodes | grep master |awk '{print $1}') 
+    kubectl annotate nodes $CONTROL_NODE external-interfaces=[ens224] --overwrite
```

Comment:
Here should it be "external-interfaces" or "externalinterfaces"? @XinShuYang @ceclinux

Reply:
It seems In K8s the words in annotation key arre separated by '-'. external-interfaces LGTM

Reply:
If this is a step needed to run multicast tests, it would be hard for a developer to run the test on their own via `go test`. Why we don't fetch it from configMap?

Reply:
This step is customized for the case with multiple outbound (external) interfaces ( the e2e case consumes this annotation to identify which node is configured with extra interfaces and requires tcpdump to check the packets forwarding). For a general setup, this step is not necessary if there is only interface.